packages/openapi-generator/src/parser/unique-naming.ts

```diff
@@ -19,9 +20,11 @@ export function ensureUniqueNames(
   namingOptions?: {
     format?: (name: string) => string;
     reservedWords?: string[];
+    separator?: string;
```

Comment:
Would it make sense to always have the same separator? Now we use it for schema names only, but file names and operation names would still suffer from the same issues.

Reply:
I added the seperator also for files and operation names.